deploy/examples/toolbox.yaml

```diff
@@ -18,9 +18,67 @@ spec:
       dnsPolicy: ClusterFirstWithHostNet
       containers:
         - name: rook-ceph-tools
-          image: rook/ceph:master
-          command: ["/bin/bash"]
-          args: ["-m", "-c", "/usr/local/bin/toolbox.sh"]
+          image: quay.io/ceph/ceph:v17.2.1
```

Comment:
@travisn what about the direct-mount and toolbox in helm charts? don't we need same changes over there also?

Reply:
Good question. My thought was to first let this settle in master and even in the 1.10 release, then see if it's requested to similarly update the other places. The pain point is that the config script is copied in so many places, so if nobody requests the change, maybe we don't change it in those other places. This request was more for development, where toolbox.yaml is used anyway.

Reply:
Sounds good. Thanks @travisn